index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Nabil's Blog</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        <div class="chom">
+            <div class="chom-triangle"></div>
```

Comment:
Instead of using HTML elements, you can try achieving this with css `::before` and `::after` selectors and a single `<div class="chimney">`. This means that your html code stays clean, free of `styling` artifacts - all appearance coding can be delegated to CSS. You'll need to get creative and add white borders to the roof, maybe. I suggesting having a play, even if just for understanding how `::before` and `::after` work!

[More reading on before and after selectors >](http://krasimirtsonev.com/blog/article/CSS-before-and-after-pseudo-elements-in-practice)